test_runner/fixtures/metrics.py

```diff
@@ -80,9 +80,6 @@ def parse_metrics(text: str, name: str = "") -> Metrics:
     "pageserver_io_operations_seconds_count",
     "pageserver_io_operations_seconds_sum",
     "pageserver_last_record_lsn",
-    "pageserver_smgr_query_seconds_bucket",
-    "pageserver_smgr_query_seconds_count",
-    "pageserver_smgr_query_seconds_sum",
```

Comment:
Missing from global metrics, probably because we don't have any compute requests in test?

Reply:
Added back in, also added a fix that all of the listed globals do not have tenant_id and timeline_id labels.